### ISSUE TITLE
BigFloat add_mul and sub_mul using MPFR

### ISSUE THREAD

**Issue by nsajko** [jump-dev/MutableArithmetics.jl#213]
After #211 it might also make sense to implement the three-argument `BigFloat` versions of the `add_mul` and `sub_mul` operations in terms of `mpfr_fma` and `mpfr_fms`.

**Comment by blegat** [comment]
Indeed, I didn't know that there were these in MPFR, it would definitely make sense
